server/api/collections.js

```diff
@@ -454,25 +454,21 @@ router.post("collections.export", auth(), async (ctx) => {
   ctx.assertUuid(id, "id is required");
 
   const user = ctx.state.user;
+  const team = await Team.findByPk(user.teamId);
+  authorize(user, "export", team);
+
   const collection = await Collection.scope({
     method: ["withMembership", user.id],
   }).findByPk(id);
-  authorize(user, "export", collection);
 
-  const filePath = await archiveCollection(collection);
+  ctx.assertPresent(collection, "Collection should be present");
+  authorize(user, "read", collection);
 
-  await Event.create({
-    name: "collections.export",
-    collectionId: collection.id,
-    teamId: user.teamId,
-    actorId: user.id,
-    data: { title: collection.title },
-    ip: ctx.request.ip,
-  });
+  exportCollections(user.teamId, user.id, user.email, id);
 
-  ctx.attachment(`${collection.name}.zip`);
-  ctx.set("Content-Type", "application/force-download");
-  ctx.body = fs.createReadStream(filePath);
+  ctx.body = {
+    success: true,
```

Comment:
Could we return the file operation as `data` here? It would allow programmatically downloading the export once it's complete if the API consumer has the `id`